app/javascript/components/widgets/twitter/tweet_body.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { fontSizes } from '../../../lib/theme';
+
+const TweetText = styled.div`
+  color: #959292;
+  font-size: ${fontSizes.large};
+  margin-bottom: 29px;
+`;
+const getStatusLink = text => {
+  const allWords = text.split(/[\s,\n]+/);
+  return allWords.filter(word => word.includes('https://t.co/'))[0];
```

Comment:
I think this could be simplified and should have a unit test. Specifically the regex could capture the first match using `https://t.co/` and your separator constraints.

Reply:
I'm not really sure, but I removed that from the history a while ago I think?

Reply:
The unit test or the regex?

Reply:
Oh this whole function. I now have it so it already does it on the back-end when getting it from the client.